### DISCUSSION TITLE
Encryption key is not valid against the schema check value ERROR

### DISCUSSION THREAD

**Discussion by vk6xebec** [authelia/authelia#4149] [Q&A] [answered]
Why would I get this error in my docker logs when trying to start up Authelia?

`
time="REMOVED" level=info msg="Authelia v4.36.9 is starting"
time="REMOVED" level=info msg="Log severity set to debug"
time="REMOVED" level=info msg="Storage schema is being checked for updates"
time="REMOVED" level=error msg="Failure running the storage provider startup check: the encryption key is not valid against the schema check value" stack="github.com/authelia/authelia/v4/internal/commands/root.go:196 doStartupChecks\ngithub.com/authelia/authelia/v4/internal/commands/root.go:84  cmdRootRun\ngithub.com/spf13/cobra@v1.5.0/command.go:876                  (*Command).execute\ngithub.com/spf13/cobra@v1.5.0/command.go:990                  (*Command).ExecuteC\ngithub.com/spf13/cobra@v1.5.0/command.go:918                  (*Command).Execute\ngithub.com/authelia/authelia/v4/cmd/authelia/main.go:10       main\nruntime/proc.go:250                                           main\nruntime/asm_amd64.s:1594                                      goexit"
time="REMOVED" level=fatal msg="The following providers had fatal failures during startup: storage" stack="github.com/authelia/authelia/v4/internal/commands/root.go:224 doStartupChecks\ngithub.com/authelia/authelia/v4/internal/commands/root.go:84  cmdRootRun\ngithub.com/spf13/cobra@v1.5.0/command.go:876                  (*Command).execute\ngithub.com/spf13/cobra@v1.5.0/command.go:990                  (*Command).ExecuteC\ngithub.com/spf13/cobra@v1.5.0/command.go:918                  (*Command).Execute\ngithub.com/authelia/authelia/v4/cmd/authelia/main.go:10       main\nruntime/proc.go:250                                           main\nruntime/asm_amd64.s:1594 
 `

This is the config.yaml section of my file that has an encryption key set (key is randomly added)

```
storage:
  encryption_key: jkfjjf933j0d8hPAi1LIhcWsZWG9qxc2rCshhsd6as997habOESXSenD5s3b # Now required
  local:
    path: /config/db.sqlite3
```

**Comment by james-d-elliott** [comment, marked as answer]
As per this previous discussion: https://github.com/authelia/authelia/discussions/4124#discussioncomment-3791753

This means the key used to initialize the database is not the key that you currently have configured. Your options are:

1. If you know the original/correct key then either:
    - Use that key in the configuration.
    - Use that key with the [CLI](http://www.authelia.com/reference/cli/authelia/authelia_storage_encryption_change-key/)  to change the storage key.
2. If you don't know the key wipe the database and start again.

**Comment by vk6xebec** [comment]
Thanks James, that was quick! I used the previous key to get out of trouble. Out of interest, if I need to wipe the database how do I find it and delete it? I looked in /var/lib for it. Apologies for the basic questions.

**Reply by james-d-elliott** [reply]
No problem. It depends on your configuration, usually if you're using sqlite with the container it's in the `/config` dir and I think it's named `db.sqlite`.

**Comment by vk6xebec** [comment]
It didn't appear that way for me. I had deleted my entire docker folder structure - all compose and config files, folders, volumes, networks, and images. When I recreated the authelia container with new keys and secrets, authelia complained about the encryption key. So it must be persisting somewhere else.
